style.css

```diff
@@ -708,3 +708,126 @@ body {
 .underline:hover {
   border: 1px solid #396df2;
 }
+
+.forms__page {
+  background-image: url(img/contact__main-back.png);
+  background-position: center;
+  background-size: cover;
+  background-repeat: no-repeat;
+  border-top-left-radius: 100px;
+  margin-top: 100px;
+}
+
+.form__wrapper {
+  background-image: url(img/contact__\ back.png);
+  background-position: center;
+  background-size: cover;
+  background-repeat: no-repeat;
+  margin: 103px 24px 0 24px;
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  justify-content: center;
+}
+
+.form__title {
+  font-family: 'Poppins', sans-serif;
+  color: #fff;
+  font-weight: 700;
+  font-style: normal;
+  font-size: 40px;
+  line-height: 52px;
+  height: 56px;
+  margin-bottom: 12px;
+}
+
+.form__title:hover {
+  text-decoration: underline;
+  text-decoration-color: #396df2;
+}
+
+.form__paragraph {
+  font-family: 'Poppins', sans-serif;
+  font-size: 20px;
+  font-style: normal;
+  line-height: 28px;
+  color: #ebebff;
+  margin: 0 24px 24px 0;
+  text-align: center;
+}
+
+.form__paragraph:hover {
+  text-decoration: underline;
+  text-decoration-color: #396df2;
+}
+
+.main__form {
+  margin: 24px 12px 30px 12px;
+  width: 100%;
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  align-items: center;
+}
+
+.input__space {
+  padding: 15px 16px;
+  font-family: 'Poppins', sans-serif;
+  margin: 0 24px 20px 24px;
+  height: 48px;
+  width: 100%;
+  color: #172b4d;
+  background: #fff;
+  border: 1px solid #cfd8dc;
+  border-radius: 8px;
+}
+
+.input__space:hover {
+  border: 1px solid #396df2;
+  text-decoration: underline;
+  text-decoration-color: #396df2;
+}
+
+.text__area {
+  padding: 15px 16px;
+  font-family: 'Poppins', sans-serif;
+  width: 100%;
+  margin: 0 24px 20px 24px;
+  background: #fff;
+  border: 1px solid #cfd8dc;
+  border-radius: 8px;
+}
+
+.text__area:hover {
+  border: 1px solid #396df2;
+  text-decoration: underline;
+  text-decoration-color: #396df2;
+}
+
+.input__btn {
+  width: fit-content;
+  height: 68px;
+  margin-right: 16px;
+  margin-top: 24px;
+  margin-bottom: 24px;
+  text-align: center;
+  align-items: center;
+  border: 1px solid #396df2;
+  border-radius: 16px;
+  background-color: white;
+  padding: 12px 12px;
+  font-family: 'Poppins', sans-serif;
+  font-size: 17px;
+  line-height: 24px;
+  color: #396df2;
+  text-decoration: none;
+  letter-spacing: 0.03em;
+  font-weight: 500;
+  align-self: flex-start;
+}
+
+.input__btn:hover {
+  border: 1px solid #396df2;
+  text-decoration: underline;
+  text-decoration-color: #396df2;
+}
```

Comment:
- [ ] Kindly add the button interactions such as hover, active (when it's pressed), disabled, and enable (default state) style as shown in the figma template.
![image](https://user-images.githubusercontent.com/43698511/134226036-1eb98f5e-918e-4469-ad89-3ee8b6015efe.png)

Reply:
- [ ] The issue here has not been properly resolved. The button's background colour doesn't change on hover, kindly fix that.

![FireShot Capture 105 - Addisalem seifu Portofolio - raw githack com](https://user-images.githubusercontent.com/46851023/134240950-68dee976-c3b9-4a01-9d32-bfda6b4a5e9b.png)